api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -53,12 +55,14 @@
     private boolean accessTokenIsJwt = true;
     private List<CertSecretSource> tlsTrustedCertificates;
     private boolean disableTlsHostnameVerification = false;
-    private boolean enableECDSA = false;
+    private boolean enableECDSA = true;
     private Integer maxSecondsWithoutReauthentication;
     private boolean enablePlain = false;
     private String tokenEndpointUri;
     private boolean enableOauthBearer = true;
     private String customClaimCheck;
+    private String useScope = null;
+    private String useAudience = null;
```

Comment:
`useScope` and `useAudience` sound to me like boolean values. Would it be better to call these just `audience` and `scope`? WDYT @tombentley?

Reply:
The idea here is that these are client side options used when the server acts as a client. So maybe the better name would thus be: `clientScope` and `clientAudience`. But that can again be interpreted to mean scope sent by client or audience sent by client (and contained in the token) to the broker. Then we could maybe call it `sendAudience` and `sendScope` which makes it clearer maybe that it's part of broker including these into the request. Or maybe call it `includeScope` and `includeAudience`.

Reply:
The only suggested alternative that does not suggest a boolean value here is `clientScope` and `clientAudience` so I renamed to that.